core/src/main/scala/chisel3/Data.scala

```diff
@@ -797,7 +797,7 @@ object Data {
   // Needed for the `implicit def toConnectableDefault`
   import scala.language.implicitConversions
 
-  private[chisel3] case class ProbeInfo(val writable: Boolean)
+  private[chisel3] case class ProbeInfo(val writable: Boolean, color: Option[layer.Layer])
```

Comment:
I'm confused. Binary compatibility _should_ be complaining about this because it isn't smart enough to filter out package private things (it looks at the Java byte code which doesn't have package private). Why are we not having to waive this? If somehow they've made MiMa support package private Scala semantics--that's great--but I'd like to understand before we merge because this is surprising me.

Reply:
This is still a mystery but we're just gonna leave it.